.devcontainer/db/scripts/admin_test_data.sql

```
@@ -0,0 +1,337 @@
+      FROM providersite ps
+      WHERE ps.provider_no = '999996'
+        AND ps.site_id = s.site_id
+  );
+
+-- Billing -------------------------------------------------------------------
+-- These cover active style listing, preview, editing, deletion, and use from
+-- Ontario service-code administration.
+INSERT INTO cssStyles (name, style, status)
+SELECT 'Local Test - Highlight', 'font-weight:bold;background-color:#fff3cd;', 'A'
+WHERE NOT EXISTS (
+    SELECT 1 FROM cssStyles WHERE name = 'Local Test - Highlight' AND status = 'A'
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)